### DISCUSSION TITLE
Melhor proposta de contrato para PJ e coisas que não devem faltar

### DISCUSSION THREAD

**Discussion by henriqueweiand** [frontendbr/forum#1356] [Discussão]
Pessoal, preciso de umas dicas aqui!

Estou iniciando a vida como PJ, e não tenho muita certeza do que seria um contrato bom para ambos os lados, empresa/profissional. Enfim, gostaria de abrir esse espaço para coletar dicas e/ou até modelos de contratos para ter uma base do que é normal em um...

Horas... valor... reajuste... duração... home office...

**Comment by Roger-Melo** [comment]
tema interessante. Também queria ter uma ideia sobre, no aguardo das possíveis interações

**Comment by mariorodeghiero** [comment]
Ta aí um assunto que também queria saber mais, estou retornando ao mercado agora em março e não faço ideia de como vai ser, se for PJ😳

**Comment by fabiochiquezi** [comment]
Trabalhei um tempo como PJ em uma empresa, mas era como se fosse a msm coisa da CLT lá, eles te cobravam horário etc... o que se for vê não poderia no contrato PJ né. Servia apenas como forma de não pagar os impostos e todos os encargos do trabalhador, e garantia uma remuneração melhor. Hoje sigo fazendo freela usando o CNPJ que fiz como PJ, mas a experiência foi boa. Mas tbm estou curioso da experiência dos outros aqui.

**Comment by feliperom** [comment]
Trabalho a um tempo como **PJ**, acredito que não se deve deixar de calcular todos os benefícios que você teria sendo **CLT**, como gastos com: Refeição, Transporte, Plano de Saúde, Valor de Previdência (mesmo sem perspectiva de aposentar rs), pois a empresa iria arcar com estes custos e fora isso acredito que estariamos pagando pra trabalhar.

Quanto a horários depende de como está acordado e sinalizado em contrato.

Até agora só tive experiências boas trabalhando como PJ.

**Comment by felipefialho** [comment]
Vale dar uma olhada #28 #106
